Drawer/Drawer.jsx

```diff
@@ -45,113 +45,42 @@ class TemporaryDrawer extends MaterialComponent {
   materialDom(props) {
     return (
       <aside
-        className="mdc-typography"
+        className="mdc-typography mdc-drawer"
         ref={control => {
           this.control = control;
         }}
         {...props}
       >
-        <nav className="mdc-temporary-drawer__drawer">{props.children}</nav>
+        <nav className="mdc-drawer__drawer">{props.children}</nav>
       </aside>
     );
   }
 }
 
-class TemporaryDrawerHeader extends MaterialComponent {
-  constructor() {
-    super();
-    this.componentName = "temporary-drawer__header";
-  }
-  materialDom(props) {
-    return (
-      <header
-        ref={control => {
-          this.control = control;
-        }}
-        {...props}
-      >
-        <div className="mdc-temporary-drawer__header-content">
-          {props.children}
-        </div>
-      </header>
-    );
-  }
-}
-
-class TemporaryDrawerContent extends MaterialComponent {
-  constructor() {
-    super();
-    this.componentName = "temporary-drawer__content";
-  }
-  materialDom(props) {
-    return (
-      <nav
-        className="mdc-list"
-        ref={control => {
-          this.control = control;
-        }}
-        {...props}
-      >
-        {props.children}
-      </nav>
-    );
-  }
-}
-
 /**
  * @prop spacer = false
  */
 class PermanentDrawer extends MaterialComponent {
   constructor() {
     super();
-    this.componentName = "permanent-drawer";
+    this.componentName = "drawer--permanent";
   }
   materialDom(props) {
     return (
-      <nav className="mdc-typography" {...props}>
-        {props.spacer && (
-          <div className="mdc-permanent-drawer__toolbar-spacer" />
-        )}
-        <div className="mdc-permanent-drawer__content">
+      <nav className="mdc-typography mdc-drawer" {...props}>
+        {props.spacer && `<div className="mdc-drawer__toolbar-spacer" />`}
```

Comment:
Is this meant to be a string here?

Reply:
oh right... stupid prettifier told me to